vlingo-e2e-sys-airline-ops/frontend/src/components/VlSelect.svelte

```diff
@@ -6,11 +6,9 @@
 
 	export let value = "";
 
-  $: {
-    if (value) {
+  $: if (value) {
```

Comment:
just realized:
`$: dispatch('change', value);`
would even be enough! Should just put that into materialify itself.